govc/cluster/module/rm.go

```diff
@@ -36,30 +40,53 @@ func init() {
 func (cmd *rm) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.ClientFlag, ctx = flags.NewClientFlag(ctx)
 	cmd.ClientFlag.Register(ctx, f)
+
+	f.StringVar(&cmd.file, "file", "", "File containing a list of module ids to delete")
 }
 
 func (cmd *rm) Usage() string {
-	return "ID"
+	return "[-file=] [ID]"
 }
 
 func (cmd *rm) Description() string {
 	return `Delete cluster module ID.
 
+When -file is specified, it reads and deletes all modules contained in the file.
+
 Examples:
-  govc cluster.module.rm module_id`
+  govc cluster.module.rm module_id
+  govc cluster.module.rm -file modules.txt`
 }
 
 func (cmd *rm) Run(ctx context.Context, f *flag.FlagSet) error {
-	if f.NArg() != 1 {
+	if f.NArg() != 1 && cmd.file == "" {
 		return flag.ErrHelp
 	}
 
-	moduleID := f.Arg(0)
-
 	c, err := cmd.RestClient()
 	if err != nil {
 		return err
 	}
+	m := cluster.NewManager(c)
+
+	if cmd.file != "" {
+		modulesFile, err := os.Open(cmd.file)
+		if err != nil {
+			return fmt.Errorf("error reading file %q: %w", cmd.file, err)
+		}
+		defer modulesFile.Close()
+
+		scanner := bufio.NewScanner(modulesFile)
+		for scanner.Scan() {
+			moduleID := scanner.Text()
+			if moduleID == "" {
+				continue
+			}
+			m.DeleteModule(ctx, moduleID)
```

Comment:
adjusted to return. But I'd be also okay with aggregating.

With aggregating it would process the next entries and maybe ddos a server which is in trouble even more?!